rtgui/rtwindow.cc

```diff
@@ -913,6 +919,14 @@ void RTWindow::writeToolExpandedStatus (std::vector<int> &tpOpen)
 }
 
 
+void RTWindow::showRawPedia()
+{
+    const Glib::ustring& uri = "https://rawpedia.rawtherapee.com/";
```

Comment:
This (accidentally?) makes use of the ["most important `const`"](https://herbsutter.com/2008/01/01/gotw-88-a-candidate-for-the-most-important-const/). :wink: